### DISCUSSION TITLE
Coinbase Advanced Trade API support

### DISCUSSION THREAD

**Discussion by netslayer** [bennycode/coinbase-pro-node#783] [Ideas]
Any plans to create a version of this for the new Coinbase Advanced Trade API?
https://docs.cloud.coinbase.com/advanced-trade-api/

**Comment by bennycode** [comment]
Coinbase Advanced Trade API is in Beta, so I wouldn't add it before its specification is final.

**Reply by netslayer** [reply]
Makes sense. They've forcibly moved (non API) users from Pro to Advanced Trade now so I assume they're be finalizing it soon ("In early 2023, all Pro API users will be migrated to Advanced Trade API").

**Comment by netslayer** [comment]
looks like they just marked the Advanced Trade API v1 out of beta. Received emails and a more aggressive popup in pro about the migration. Haven't dived too deep into how different the calls are but data returned looks similar, any ideas on LOE to port?

**Comment by SethTurin** [comment]
I wanna just add my voice in here to gently nudge this cause

**Reply by bennycode** [reply]
Happy to see your interest. Is the current API not working anymore?

**Reply by SethTurin** [reply]
Very grateful for your work on this project.

The current API is still working, but people are actively being discouraged from using coinbase pro in favor of coinbase advanced trading. My use case involves passing a wrapped version of the api functionality along to non-technical users. I've had one or two of them ask me if there were plans to update to coinbase advanced pro.

**Comment by georgewwindsor** [comment]
bump,  is this still being worked on? Coinbase now is automatically moving peoples assets to their new api...all my crypto got moved... i cant add new funds either.

**Comment by theimpostor** [comment]
I am getting this error trying to use the API with 9.0.0 of this lib:

```
Error: Trading disabled: your Pro account is being migrated/has been migrated to Advanced Trade. Please visit coinbase.com/advanced-trade to trade with the same low fees as pro.
```

It used to work for many years 😢

**Reply by bennycode** [reply]
Yikes! It seems like all v2 endpoints have to be migrated to v3 in order to make it compatible: https://docs.cloud.coinbase.com/advanced-trade-api/docs/rest-api-pro-mapping
